packages/tools/kadena-cli/src/account/commands/accountFund.ts

```diff
@@ -126,15 +127,40 @@ export const createAccountFundCommand = createCommand(
     //   }
     // }
 
-    const loader = ora('Funding account...\n').start();
-
     const result = await fund(config);
-    assertCommandError(result, loader);
+    assertCommandError(result);
 
     log.info(
       log.color.green(
-        `"${accountConfig.name}" account funded with "${amount}" ${accountConfig.fungible} on chain ${chainId} in ${networkConfig.networkId} network.`,
+        `Explorer URL: ${networkConfig.networkExplorerUrl}\nRequest Key: ${result.data.requestKey}`,
```

Comment:
I think this url should be including the request key:

`https://explorer.chainweb.com/testnet/tx/${result.data.requestKey}`